spring-cloud-stream-binder-rocketmq/src/main/java/com/alibaba/cloud/stream/binder/rocketmq/RocketMQBinderUtils.java

```diff
@@ -38,7 +42,7 @@ public static RocketMQBinderConfigurationProperties mergeProperties(
 			result.setNameServer(rocketBinderConfigurationProperties.getNameServer());
 		}
 		else {
-			result.setNameServer(rocketMQProperties.getNameServer());
+			result.setNameServer(Arrays.asList(rocketMQProperties.getNameServer()));
```

Comment:
`rocketMQProperties.getNameServer()` is separated by ';' and couldn't be handle by `Arrays.asList` .

Reply:
thanks, i fixed it